ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -0,0 +1,94 @@
+#![allow(unused)]
+use crate::{crypto::hash, ssz::ByteVector};
+use alloy_primitives::{uint, U256};
+use blst::min_pk::PublicKey;
+pub use c_kzg::{Bytes32, Bytes48, Error, KzgSettings};
```

Comment:
`pub` here makes everything you import in this line visible.. is this what you want?

Reply:
Nope